deploy/roles/python3rh7/tasks/main.yml

```diff
@@ -2,21 +2,24 @@
 ---
 - name: Install Python36 packages
   yum:
-    name: "{{ item }}"
+    name: 
+      - python36
+      - python36-PyYAML
+      - python36-setuptools
+      - python36-devel
+      - python36-libs
+      - python36-tkinter
+      - python36-numpy
+      - python36-pip
+      - python36-tools
+      - rh-python36
+      - rh-python36-numpy
+      - rh-python36-scipy
+      - rh-python36-python-pip
+      - rh-python36-python-tools
+      - rh-python36-python-six
+      - rh-python36-python-virtualenv
     state: present
-  with_items:
-    - python36
-    - python36-PyYAML
-    - python36-setuptools
-    - python36-devel
-    - python36-libs
-    - python36-tkinter
-    - rh-python36-numpy
-    - rh-python36-scipy
-    - rh-python36-python-pip
-    - rh-python36-python-tools
-    - rh-python36-python-six
-    - rh-python36-python-virtualenv
 
 #- name: Pip install virtualenvwrapper
 #  pip: executable=pip3
```

Comment:
We should add a section below to upgrade pip for the EPEL version of Python 36 as well:
```
- name: Upgrade Pip
  pip:
    name: pip
    executable: pip3
    umask: '0022'
```